sympy/core/tests/test_var.py

```diff
@@ -1,83 +1,91 @@
 # Tests for var are in their own file, because var pollutes global namespace.
-
+import textwrap
 from sympy import Symbol, var, Function, FunctionClass
 from sympy.utilities.pytest import raises
 
 # make z1 with call-depth = 1
```

Comment:
The comment `# make z1 with call-depth = 1` belonged to the `_make_z1` function but now it's lonely :( Either move it to the function it describes or delete it.